pkg/ocm/placementrule.go

```diff
@@ -26,10 +26,47 @@ type PlacementRuleBuilder struct {
 	errorMsg string
 }
 
+// NewPlacementRuleBuilder creates a new instance of PlacementRuleBuilder.
+func NewPlacementRuleBuilder(apiClient *clients.Settings, name, nsname string) *PlacementRuleBuilder {
+	glog.V(100).Infof(
+		"Initializing new placement rule structure with the following params: name: %s, nsname: %s",
+		name, nsname)
+
+	builder := PlacementRuleBuilder{
+		apiClient: apiClient,
+		Definition: &placementrulev1.PlacementRule{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: nsname,
+			},
+		},
+	}
+
+	if name == "" {
+		glog.V(100).Info("The name of the PlacementRule is empty")
+
+		builder.errorMsg = "placementrule's 'name' cannot be empty"
+	}
+
+	if nsname == "" {
+		glog.V(100).Info("The namespace of the PlacementRule is empty")
+
+		builder.errorMsg = "placementrule's 'nsname' cannot be empty"
+	}
```

Comment:
```suggestion
	if name == "" {
		glog.V(100).Info("The name of the PlacementRule is empty")

		builder.errorMsg = "placementrule's 'name' cannot be empty"
		
		return builder
	}

	if nsname == "" {
		glog.V(100).Info("The namespace of the PlacementRule is empty")

		builder.errorMsg = "placementrule's 'nsname' cannot be empty"
		
		return builder
	}
```

Reply:
@klaskosk After speaking with @kononovn we decided your original approach was fine for now.  If you want to revert my above suggestion that would be great.